### ISSUE TITLE
support Cassandra 5 / trunk with AxonOps

### ISSUE THREAD

**Issue by rustyrazorblade** [rustyrazorblade/easy-cass-lab#80]
The dashboards were really helpful for fixing [CASSANDRA-19477](https://issues.apache.org/jira/browse/CASSANDRA-19477), let's make it work with C* 5.0 branch, beta1 and trunk (5.1).

**Comment by rustyrazorblade** [comment]
```
sudo yum install axon-cassandra5.0-agent
sudo apt-get install axon-cassandra5.0-agent
```

**Comment by jrwest** [comment]
We would need to tweak the logic to use the version from the axonops key in `cassandra_versions.yaml` but this should be relatively straightforward to add if we know the version works with the specific C* version

**Comment by jrwest** [comment]
And if want to use the same axonops version for multiple C* versions we need to extend the logic a bit to skip

**Comment by rustyrazorblade** [comment]
For the duplicates I think this will work nicely: https://mikefarah.gitbook.io/yq/operators/unique

**Comment by rustyrazorblade** [comment]
Merged #81
